rules/windows/defense_evasion_masquerading_as_svchost.toml

```diff
@@ -0,0 +1,92 @@
+[metadata]
+creation_date = "2025/11/12"
+integration = ["windows"]
+maturity = "production"
+updated_date = "2025/11/12"
```

Comment:
https://www.elastic.co/docs/reference/query-languages/esql/functions-operators/operators#esql-match-operator

May need to add min-stack for 9.1.0.